### ISSUE TITLE
Bxml2Reader breaks 

### ISSUE THREAD

**Issue by lg188** [kytulendu/gw2formats#2]
I couldn't build gw2formats so I tried to bisect it. 48faaa4659e6526db3288e15ff1008fa90f762b9 is the commit that stops me from compiling.
I followed the instructions from gw2browser. 

this is the error that I get: 
```
[  1%] Building CXX object CMakeFiles/gw2formats.dir/src/bxml2/Bxml2Reader.cpp.o
/home/lg188/repo/Gw2Browser/extern/gw2formats/src/bxml2/Bxml2Reader.cpp: In member function ‘void gw2f::bxml2::Bxml2Reader::populateXmlDocument(gw2f::bxml2::XmlDocument&) const’:
/home/lg188/repo/Gw2Browser/extern/gw2formats/src/bxml2/Bxml2Reader.cpp:128:59: error: ‘::_itoa’ has not been declared
  128 |                 element->setAttribute( "__numChildren", ::_itoa( numChildren, buffer, 10 ) );
      |                                                           ^~~~~
make[2]: *** [CMakeFiles/gw2formats.dir/build.make:104: CMakeFiles/gw2formats.dir/src/bxml2/Bxml2Reader.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:84: CMakeFiles/gw2formats.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

**Comment by kytulendu** [comment]
You need to update the gw2formats to current revision. It already fixed in 282e34b.

**Comment by kytulendu** [comment]
Close the issue.
